test/types/string/dmk/oobcodepoint.suppressif

```diff
@@ -0,0 +1,11 @@
+# --fast sets --no-checks
+#
+# --fast sets --no-checks and this test is explicitly checking that one
+# of our runtime checks is working correctly. There's been debate as to
+# whether tests like this should be suppressed or skipped. For now we
+# suppress with the argument that we're testing that --fast actually
```

Comment:
I don't think this argument actually explains why you should suppressif it. I think you should skipif it. I think we should use suppressif for cases that we expect to be resolved on their own at some point (e.g. backend C compiler issues) and .futures for things that will only resolve with some effort on our part. But this case seems to be neither of those - I don't have any expectation that `--fast` will ever stop throwing `--no-checks`.

All that said it doesn't bother me if you want to keep this a .suppressif for some reason. I just don't understand the reason to do so from what is written here.

Reply:
Actually, I don't care which way it goes myself.  I copied this suppressif file byte-for-byte from one of Brad's, which is how I decided to suppress instead of skip.  I think I'll leave it as a suppression only because it came from Brad.  I don't mind if the whole issue gets decided differently later, in which case several suppressif files in various parts of the tests will need to be changed to skipifs.